roles/tas_single_node/tasks/podman/restore.yml

```diff
@@ -0,0 +1,80 @@
+- name: Create /tmp/restore_mode_active
+  ansible.builtin.file:
+    path: /tmp/restore_mode_active
+    state: touch
+    mode: "0644"
+
+- name: Extract Basename from File
+  ansible.builtin.set_fact:
+    restore_file: "{{ tas_single_node_data_protection.restore.source | basename }}"
+
+- name: Push Encrypted Backup Tarball to Remote Server
+  ansible.builtin.copy:
+    src: "{{ tas_single_node_data_protection.restore.source }}.tar.gz.enc"
+    dest: "/root/{{ restore_file }}.tar.gz.enc"
```

Comment:
`/root/` is home directory of root; I'd suggest doing this in a dedicated subdirectory of `tas_single_node_data_protection.backup.directory`? You could name it same as the archive (without the file extension) to ensure certain degree of uniqueness of the name, too.